include/deal.II/grid/grid_out.h

```diff
@@ -865,7 +865,31 @@ namespace GridOutFlags
    * @ingroup output
    */
   struct Vtk : public DataOutBase::VtkFlags
-  {};
+  {
+    /**
+     * Default constructor.
+     */
+    Vtk()
+      : draw_cells(true)
+      , draw_faces(true)
+      , draw_co_faces(true)
+    {}
+
+    /**
+     * Draw cells.
+     */
+    bool draw_cells;
```

Comment:
I think `output_cells` etc would be a more appropriate name. Nothing is being drawn by this class :-)